templates/app/templates/frontend/src/store/index.js

```diff
@@ -83,30 +73,13 @@ export default new Vuex.Store({
       commit("accountUpdate", { account });
     },
     async entitySubmit({ state }, { type, body }) {
-      const accountURL = `${API}/auth/accounts/${state.client.senderAddress}`;
-      const account = (await axios.get(accountURL)).data.result.value;
-      const accountNumber = account.account_number;
-      const sequence = account.sequence;
-      const address = state.client.senderAddress;
-      const chain_id = await state.client.getChainId();
-      const req = {
-        base_req: { chain_id, from: address },
-        creator: address,
-        ...body,
-      };
+      const { chain_id } = state;
+      const creator = state.client.senderAddress;
+      const base_req = { chain_id, from: creator };
+      const req = { base_req, creator, ...body };
       const { data } = await axios.post(`${API}/${chain_id}/${type}`, req);
       const { msg, fee, memo } = data.value;
```

Comment:
Any chance to rename this field to `msgs` in the data source (since it is a list)?

This one `msg` field [someone forgot to rename to `msgs` when it was changed to a list in Cosmos SDK's StdTx](https://github.com/cosmos/cosmos-sdk/blob/v0.39.0/x/auth/types/stdtx.go#L26) is a big pain. We should not copy that.

Reply:
We are using `WriteGenerateStdTxResponse` [1], which writes `StdTx`, so unless we replace response writer with our own (I would rather we didn't), then we're stuck with msg/msgs.

[1] https://github.com/cosmos/cosmos-sdk/blob/v0.39.0/x/auth/client/utils/rest.go#L15

Reply:
I see, thank you. Then we have to live with it until stargate is out.